contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,303 @@ availability:
+```php
+PostHog::capture(array(
+  'distinctId' => 'distinct_id',
+  'event' => 'event_name_of_your_goal_metric',
+  'properties' => array(
+    '$feature/experiment_feature_flag_name'  = 'variant_name'
+  )
+));
+```
 
-Experimentation uses multivariate Feature Flags under the hood to handle user allocation to an experiment. You'll notice this when checking if a user is assigned to an experiment.
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id',
+    event: 'event_name_of_your_goal_metric',
+    properties: {
+        '$feature/experiment_feature_flag_name': 'variant_name',
+    }
+})
+```
 
-While Feature Flags can be boolean or multivariate, Experimentation always uses a multivariate approach.
+```go
+client.Enqueue(posthog.Capture{
+  DistinctId: "distinct_id",
+  Event:      "event_name_of_your_goal_metric",
+  Properties: posthog.NewProperties().
+    Set("$feature/experiment_feature_flag_name", "variant_name"),
+})
+```
+
+```java
+posthog.capture(
+  "distinct_id",
+  "event_name_of_your_goal_metric",
+  new HashMap<String, Object>() {
+    {
+      put("$feature/experiment_feature_flag_name", "variant_name");
+    }
+  }
+);
+```
+
+</MultiLanguage>
+
+## Launching your experiment
+
+Once you've written your code, it's a good idea to test that each variant behaves as you'd expect. If you find out your implementation had a bug after you've launched the experiment, you lose days of effort as the experiment results can no longer be trusted.
+
+Once you're satisfied, you're ready to launch your experiment.
+
+> **Note:** The feature flag is activated only when you launch the experiment.
+
+> **Note:** While the [PostHog toolbar](/docs/product-analytics/toolbar) enables you to toggle feature flags on and off, this only works for active feature flags and won't work for your experiment feature flag while it is still in draft mode.
+
+## Viewing experiment results
+
+While the experiment is running, you can see results on the page:
+
+![Viewing experiment results](../../images/features/experiments/experiment-results.png)
+
+Sometimes, in the beginning of an experiment, results can be skewed to one side while enough data is being still gathered. While peeking at preliminary results is not a problem, making quick decisions based on them is [problematic](/blog/ab-testing-mistakes#3-conducting-an-experiment-without-a-predetermined-duration). We advise you to wait until your experiment has had a chance to gather enough data.
+
+## Ending an experiment
+
+Here is checklist of things to do when ready to end your experiment:
+-   [ ] Click on "Stop" button on the experiment page. This will ensure final results are stored.
+-   [ ] Remove your experiment code and launch the winning variant to all your users.
```

Comment:
Can you rollout the winning feature flag to 100% of users after an experiment is done? We should recommend doing that before removing the code.

Reply:
There's a banner recommending this once you end the experiment. Indeed we should.